sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -90,7 +90,8 @@ trait BinaryArrayExpressionWithImplicitCast extends BinaryExpression
       > SELECT _FUNC_(NULL);
        -1
   """,
-  since = "1.5.0")
+  since = "1.5.0",
+  group = "array_funcs")
 case class Size(child: Expression, legacySizeOfNull: Boolean)
   extends UnaryExpression with ExpectsInputTypes {
```

Comment:
`Size` is one of the more problematic ones - should it be an array_func or a map_func or better yet both?

Reply:
hm, I see. Another idea: how about making a group for more general cases, e.g., `collection_funcs`? cc: @HyukjinKwon

Reply:
Seems like `size` is `collection_funcs` in `functions.scala`. I think we can just stick to it.